### PR TITLE
Add new response for correct and incorrect answer's

### PR DESCRIPTION
**Pull request by humsarika**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/modal-responses.ts [hunk 1]
```diff
@@ -13,4 +14,6 @@ export const incorrectModalResponses = [
   "Incorrect. Better luck next time.",
   "Unfortunately, your answer was incorrect.",
   "Oops! That is incorrect.",
-];
+  "Incorrect; Try Again",
```
Comment (jdwilkin4) [inline src/data/modal-responses.ts:17]:
```suggestion
  "Incorrect, try again",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -841,6 +841,16 @@ const linuxQuiz = [
...
+  {
+    Question: "Which of the following is NOT a category of file permissions in Linux?",
+    Answer: "compile",
+    Distractor1: "execute",
+    Distractor2: "write",
+    Distractor3: "read",
+    Explanation:
+      "In Linux, file permissions fall into three categories: read, write and execute.",
+    Link: "https://www.freecodecamp.org/news/linux-chmod-chown-change-file-permissions/",
+  },
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:853]: This is a good addition.

Can you also remove this same question from the `security` quiz. 
Unfortunately it was miscategorized when we split up the quiz files.

You will probably find it closer to the top of the `security` quiz file. 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/linux-quiz.ts (hunk 2 above)
Comment (humsarika) [reply src/data/linux-quiz.ts:853]: Sure , I will. @jdwilkin4

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**